core/backend/src/rpc-impl/IModelWriteRpcImpl.ts

```diff
@@ -84,11 +84,11 @@ export class IModelWriteRpcImpl extends RpcInterface implements IModelWriteRpcIn
   }
 
   public async hasUnsavedChanges(tokenProps: IModelRpcProps): Promise<boolean> {
-    return IModelDb.findByKey(tokenProps.key).txns.hasUnsavedChanges;
+    return BriefcaseDb.findByKey(tokenProps.key).txns.hasUnsavedChanges;

```

Comment:
Here and elsewhere you switched from `IModelDb` to `BriefcaseDb`. What about `StandaloneDb`, which also has `txns` and permits editing?

Reply:
I believe that every place I (now) use BriefcaseDb is correct